workflows/files/ZWEWRF06.xml

```diff
@@ -0,0 +1,251 @@
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>false</canMarkAsFailed>
+        <rest>
+            <httpMethod>PUT</httpMethod>
+            <uriPath substitution="true">/zosmf/restconsoles/consoles/${_step-stepOwnerUpper}</uriPath>
+            <requestBody substitution="true">#if(${instance-xmem_dasd.startsWith("STORCLAS")})
+#set($dasd = "SMS")
+#elseif(${instance-xmem_dasd.startsWith("VOL=SER=")})
+#set($dasd = "VOLUME=${instance-xmem_dasd.substring(8)}")
+#end
+{"cmd":"SETPROG APF,ADD,DSNAME=${instance-xmem_prefix}.LOADLIB,${dasd}"}</requestBody>
+            <expectedStatusCode>200</expectedStatusCode>
+            <!-- REST mappings -->
+        </rest>
+    </step>
+    <step name="modify_scheduler" optional="false">
+        <title>Modify Scheduler</title>
+        <description>Modifies the current SCHEDnn PARMLIB member
+to run XMEM Server to run in key 4 and be non-swappable</description>
+        <prereqStep name="copy_artifacts"/>
+        <!--variableValues-->
+        <!-- condition -->
+        <instructions substitution="false">Run this step to modify the SCHEDnn PARMLIB member</instructions>
+        <weight>1</weight>
+        <skills>z/OS System Programmer</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>false</canMarkAsFailed>
+        <template>
+            <inlineTemplate substitution="true">//CONFIGSH EXEC PGM=IEBGENER
+//SYSPRINT DD SYSOUT=*
+//SYSIN    DD DUMMY
+//SYSUT1   DD DISP=SHR,
+//    DSN=${instance-parmlib}(${instance-sched})
+//         DD DISP=SHR,
+//    DSN=${instance-zowe_dsn_prefix}.SZWESAMP(ZWESISCH)
+//SYSUT2   DD DISP=SHR,
+//    DSN=${instance-parmlib}(${instance-sched})</inlineTemplate>
+            <submitAs maxRc="0">JCL</submitAs>
+            <maxLrecl>1024</maxLrecl>
+            <!-- zosmfOutput -->
+        </template>
+    </step>
+    <step name="reload_scheduler" optional="false">
+        <title>Reload Scheduler Settings</title>
+        <description>This step will make the previous changes effective immediately</description>
+        <prereqStep name="modify_scheduler"/>
+        <!--variableValues-->
+        <!-- condition -->
+        <instructions substitution="false">Run this step to reload the scheduler settings</instructions>
+        <weight>1</weight>
+        <skills>z/OS System Programmer</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>false</canMarkAsFailed>
+        <rest>
+            <httpMethod>PUT</httpMethod>
+            <uriPath substitution="true">/zosmf/restconsoles/consoles/${_step-stepOwnerUpper}</uriPath>
+            <requestBody substitution="true">#if(${instance-sched.startsWith("SCHED")})
+#set($SCH_ID = '${instance-sched.substring(5)}')
+{"cmd":"SET SCH=${SCH_ID}"}
+#end</requestBody>
+            <expectedStatusCode>200</expectedStatusCode>
+            <!-- REST mappings -->
+        </rest>
+    </step>
+    <step name="start_xmem" optional="false">
```

Comment:
add an optional (?) step to update PARMLIB(COMMANDxx) so ZWESISTC is started automatically at IPL.